docs/mixins/child-apps.md

```diff
@@ -222,3 +224,55 @@ myApp.removeChildApps();
 ```
 
 This will destroy all childApps (that don't have preventDestroy set to true), and remove them.
+
+### ChildAppsMixin `startChildApp`
+
+You can quickly start a specific childApp from an
+App instance by calling the `startChildApp`
+method and passing the childApp name and any options.
+
+```js
+var childApps = {
+   cA1: Marionette.Toolkit.App.extend({
+     onStart(options) {
+       this.mergeOptions(options, ['foo']);
+     }
+   })
+};
+
+var myApp = new Marionette.Toolkit.App({ childApps: childApps });
+
+myApp.startChildApp('cA1', { foo: 'bar' });
+
+var childAppInstance = myApp.getChildApp('cA1');
+
+// true
+console.log(childAppInstance.isRunning());
+
+// bar
+console.log(childAppInstance.getOption('foo'));
+```
+
+Note: The childApp instance is returned for chaining.
```

Comment:
ahh except it isn't.  it's returning `myApp`.. which after thinking about it is probably what you want.. but it isn't the childApp